packages/broker/test/integration/plugins/operator/OperatorPlugin.test.ts

```diff
@@ -56,7 +56,7 @@ describe('OperatorPlugin', () => {
         })
         // wait for a while so that MaintainTopologyService has time to handle addStakedStreams
         // events emitted during Broker start
-        await wait(500)
+        await wait(5000)
```

Comment:
Does this fix test flakyness? Would probably make sense to somehow wait for the broker to have subscribed to the stream